content/patterns/button/examples/button.html

```diff
@@ -195,6 +195,18 @@ <h2 id="sc1_label">HTML Source Code</h2>
           sourceCode.make();
         </script>
       </section>
+
+
+      <section>
+        <h2>Support Levels</h2>
+        <iframe
```

Comment:
Need an H3 here for "Command Button"

Reply:
Addressed in https://github.com/w3c/aria-at-app/pull/479.